### ISSUE TITLE
Provide `script/train.sh`

### ISSUE THREAD

**Issue by yjoonjang** [mangopy/tool-retrieval-benchmark#1]
Hi, @mangopy. I found your tool retrieval work interesting. Nice work on this!
I have some questions:

1. It looks like `script/train.sh` is not provided. Could you please upload the training script?
2. Also, I am questionable what the `train_data` is from [https://huggingface.co/datasets/mangopy/ToolRet-Queries](https://huggingface.co/datasets/mangopy/ToolRet-Queries). It doesn't look to be splitted. Could you provide some details for me ?

**Comment by mangopy** [comment]
Thank you for your attention and acknowledgment of our work.

1. I have uploaded the `script` folder. You can check the repo again.
2. In terms of the `train_data`, we have not released it currently. We decide to release it after the peer-reviewing period. And I will update this GitHub issue once we release it. The ToolRet-Queries (https://huggingface.co/datasets/mangopy/ToolRet-Queries) is the evaluation benchmark (test set). All the data from the ToolRet-Queries is used for evaluation.

**Comment by yjoonjang** [comment]
Very nice of you. Thank you so much!
